lib/grandpa/message_tracker_test.go

```diff
@@ -180,3 +181,50 @@ func TestMessageTracker_MapInsideMap(t *testing.T) {
 	_, ok = voteMsgs[authorityID]
 	require.True(t, ok)
 }
+
+func TestMessageTracker_handleTick(t *testing.T) {
+	kr, err := keystore.NewEd25519Keyring()
+	require.NoError(t, err)
+
+	gs, in, _, _ := setupGrandpa(t, kr.Bob().(*ed25519.Keypair))
+	gs.tracker = newTracker(gs.blockState, gs.messageHandler)
+
+	msg := &VoteMessage{
+		Round: 100,
+	}
+	gs.tracker.addVote(&networkVoteMessage{
+		msg: msg,
+	})
+
+	gs.tracker.handleTick()
+
+	select {
```

Comment:
nit define it inlined

```suggestion
	const testTimeout = time.Second
	select {
```

Reply:
updated